### ISSUE TITLE
Help menu text fields not updating on language file reload

### ISSUE THREAD

**Issue by winnipeg21** [TownyAdvanced/Towny#4220]
### What steps will reproduce the problem?
1. Change any message in the language file that is part of a help menu (example changing` /town`'s title
2. Doing `/townyadmin reload lang` doesn't update that message

### What is the expected output?
1. `/townyadmin reload lang` should update every message field

### What is your Towny version?
Do not say "latest".
0.96.2.8

### What is your spigot/craftbukkit version?
Do not say "latest".
Paper #116

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/tkgu5FBP
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/uRy8ZLMh
3. Your townyperms.yml : https://pastebin.com/wu8u50gq
4. Your log's error :  No error in console really

Info (make my own section cause i'm a boss)

Basically if you example you edit the `help_0:` field in your language file to lets say `&4&lGeneral Help` you'd think by doing `/townyadmin reload lang` it would update that message to the right name/color. Well it sadly doesn't, to make it update you need to reload your server or restart it. TO BE NOTED: I am using a custom language file so no overwriting happens. I've only tested this with the first 4 help text fields.

**Comment by LlmDl** [comment]
So I've found out why this is occuring.

We store many of the help menus in static final array lists (which don't get touched again when the lang file is reloaded.)

Can probably work out something.

**Comment by winnipeg21** [comment]
![Capture](https://user-images.githubusercontent.com/9980707/88570029-bfda5600-d011-11ea-9ace-52a1a2bb42fc.PNG)
Hmm 🤔 Ight thanks for the help mate!
